static/js/init.js

```diff
@@ -76,12 +76,12 @@ $(document).ready(function() {
     if ( $pageControls.length > 0 ) {
         var scrollPos,
             controlsHeight,
-            controlsTop = $pageControls.offset().top + 100;
+            controlsTop = $pageControls.offset().top - 41;
```

Comment:
Will add a note here, and I agree, I'd like to move this into it's own module as well. It will depend on what we end up doing for the sticky side menu.